### PR TITLE
docs: add in-cluster redis backup document

### PR DESCRIPTION
**Pull request by chlins**
Close: https://github.com/goharbor/harbor-operator/issues/566

Signed-off-by: chlins <chlins.zhang@gmail.com>

### CODE REVIEW

File: docs/LCM/backup-redis-data.md [hunk 1]
```diff
@@ -2,4 +2,51 @@
...
+data:
+  password: dGZsaWlqeGo=
+  redis-password: dGZsaWlqeGo=
+kind: Secret
+metadata:
+  name: harborcluster-sample-redis
+  namespace: cluster-sample-ns
+  ......
+
+# echo dGZsaWlqeGo=|base64 -d
```
Comment (steven-zou) [inline docs/LCM/backup-redis-data.md:24]: What about 

```bash
kubectl get secret harborcluster-sample-redis -n cluster-sample-ns -o jsonpath='{.data.password}' | base64 --decode
```

File: docs/LCM/backup-redis-data.md [hunk 2]
```diff
@@ -2,4 +2,51 @@
...
+Warning: Using a password with '-a' or '-u' option on the command line interface may not be safe.
+OK
+```
+
+## Step 3
+
+Copy redis dump.rdb from redis pod to local path.
+
+```bash
+# kubectl cp cluster-sample-ns/rfr-harborcluster-sample-redis-0:dump.rdb dump.rdb
```
Comment (steven-zou) [inline docs/LCM/backup-redis-data.md:49]: Is there any way to directly dump the data to the host? I'm wondering if it is ok when the data size is big.

**Review by steven-zou — CHANGES_REQUESTED** [review]
Please do changes

File: docs/LCM/backup-redis-data.md (hunk 1 above)
Comment (chlins) [reply docs/LCM/backup-redis-data.md:24]: Good idea.

File: docs/LCM/backup-redis-data.md (hunk 2 above)
Comment (chlins) [reply docs/LCM/backup-redis-data.md:49]: How about let user backup redis pvc directly?

File: docs/LCM/backup-redis-data.md [hunk 3]
```diff
@@ -2,4 +2,37 @@
 
 Documentation shown here guides you how to backup the Redis data.
 
-TBD
+## Step 1
+
+Get redis password from harborcluster namespace.
+
+```bash
+# kubectl get secret harborcluster-sample-redis -n cluster-sample-ns -o jsonpath='{.data.password}' | base64 --decode
```
Comment (steven-zou) [inline docs/LCM/backup-redis-data.md:10]:
```suggestion
kubectl get secret harborcluster-sample-redis -n cluster-sample-ns -o jsonpath='{.data.password}' | base64 --decode
```

File: docs/LCM/backup-redis-data.md [hunk 4]
```diff
@@ -2,4 +2,37 @@
...
+tfliijxj
```
Comment (steven-zou) [inline docs/LCM/backup-redis-data.md:11]:
```suggestion
#tfliijxj
```

**Review by steven-zou — CHANGES_REQUESTED** [review]
please do changes

**Review by steven-zou — APPROVED** [review]
LGTM

**Review by bitsf — APPROVED** [review]

**Pull request merged**